VAMobile/src/screens/HealthScreen/SecureMessaging/EditDraft/EditDraft.tsx

```diff
@@ -511,26 +511,26 @@ function EditDraft({ navigation, route }: EditDraftProps) {
   function renderAlert() {
     return (
       <Box my={theme.dimensions.standardMarginBetween}>
-        <AlertBox border={'warning'} title={t('secureMessaging.reply.youCanNoLonger')}>
+        <AlertWithHaptics variant="warning" header={t('secureMessaging.reply.youCanNoLonger')}>
           <TextView mt={theme.dimensions.standardMarginBetween} variant="MobileBody">
             {t('secureMessaging.reply.olderThan45Days')}
```

Comment:
Could potentially be simplified moving this to the `description` prop to AlertWithHaptics.